crossbar/router/auth/pending.py

```diff
@@ -45,27 +72,28 @@ class PendingAuth:
 
     AUTHMETHOD = 'abstract'
 
-    def __init__(self, session, config):
+    def __init__(self, pending_session_id, transport_info, realm_container, config):
```

Comment:
yeah, perfect! so this removes the (deep) session object reference, which was the source of the strong binding to routing internals the authenticator actually does not need